tests/testthat/test-bundles.R

```diff
@@ -6,31 +6,31 @@ library(testthat)
 test_that("available_bundles isn't empty and have correct columns", {
   result <- omopbundles::available_bundles()
   expect_true(nrow(result) > 0, info = "The dataframe should not be empty")
-  hiv_ab <- filter(result, concept_name == "antibodies to hiv")
+  hiv_ab <- filter(result, concept_name == "Antibodies to HIV")
```

Comment:
```suggestion
  hiv_ab <- dplyr::filter(result, concept_name == "Antibodies to HIV")
```

Make sure we use `dplyr`'s filter and not `stats::filter`